src/coreclr/jit/compiler.h

```diff
@@ -1798,13 +1798,17 @@ class FlowGraphDfsTree
     // Whether the DFS that produced the tree used edge likelihoods to influence successor visitation order.
     bool m_profileAware;
 
+    // Whether the DFS reflects Wasm control flow rules.
+    bool m_forWasm;
```

Comment:
What's the scenario where we would want m_forWasm==true for non-wasm targets, or m_forWasm==false for wasm targets?